test/DisplayBoardTests.cpp

```diff
@@ -0,0 +1,85 @@
+//added for testing private methods
+//acccording to google test guide
+#include "DisplayBoard.cpp"
```

Comment:
This will be fixed in next commit:
https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#testing-private-code

Reply:
Why in next one? I am 100% sure that this can be done in this one and still be in scope of task.

Reply:
Next commit, not task ;)

Reply:
done

Reply:
Still not fixed...